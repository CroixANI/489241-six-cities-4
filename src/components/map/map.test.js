import React from 'react';
import renderer from 'react-test-renderer';

import Map from './map';

const LOCATIONS = [
  {latitude: 52.3909553943508, longitude: 4.85309666406198},
  {latitude: 52.369553943508, longitude: 4.85309666406198},
  {latitude: 52.3909553943508, longitude: 4.929309666406198},
  {latitude: 52.3809553943508, longitude: 4.939309666406198},
];

it(`Render Map component`, () => {
  const tree = renderer
    .create(<Map locations={LOCATIONS} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
