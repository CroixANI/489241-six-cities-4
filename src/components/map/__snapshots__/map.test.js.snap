// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Map component 1`] = `
<section
  className="cities__map map"
/>
`;
